upload/modules/CraftingStore/module.php

```diff
@@ -71,16 +71,7 @@ public function __construct(
         $pages->add('CraftingStore', '/panel/craftingstore/sync', 'resources/view/panel/sync.php');
     }
 
-    /**
-     * @var User $user
-     * @var Pages $pages
-     * @var Cache $cache
-     * @var Smarty $smarty
-     * @var Navigation[] $navs
-     * @var Widget $widgets
-     * @var null $template
-     */
-    public function onPageLoad($user, $pages, $cache, $smarty, $navs, $widgets, $template)
+    public function onPageLoad(User $user, Pages $pages, Cache $cache, Smarty $smarty, $navs, Widgets $widgets, $template)
```

Comment:
Some of the items in this list do not have types, and the doc has been removed. For methods that have items that cannot have a (correct) typehint (e.g. an array filled with a specific object) I would like to still have the normal doc-block so the idea knows what to do with it.

I also see that some do not have a typehint, why is that?